src/components/EpisodesNav/styles.scss

```
@@ -1,14 +1,120 @@
-.paginationLink {
-	font-size: 1em; //placeholder to have the classname showing in inspector
+.wrapper {
+	position: fixed; //with absolute it will scroll with page
+	width: 80%;
+	left: 50%;
+	bottom: 5%;
+	transform: translateX(-50%);
 }
 
-.navButton {
-	width: 50px;	//placeholder so they are visible
-	height: 50px;
-	margin: 5px;
+	.container {
+		display: flex;
+		justify-content: space-between;
+		align-items: center;
+	}
+
+		.button {
+			font-size: 1.75rem;
+			width: 1em;	
+			height: 1.42em;
+
+			&[aria-hidden="true"] {
+				pointer-events: none; //disable and hide hidden
+				opacity: 0;
+			}
+		}
+
+		.prev {
+			order: 1;
+		}
+
+		.next {
+			order: 3;
+		}
+
+		.pagination {
+			order: 2;
+			display: flex;
+			width: 40%;
+			height: 1.5em;
+			align-items: center;
+			list-style: none;
+		}
+
+			.inactive, .active {
```

```suggestion
			.item {
```

aaah, here's the fellah! yeah, let's just use the 'rest' state for this one with a separate class name - that way we can do `item.active` and `item.inactive` which reads better imho